cmd/pingdom-exporter/main.go

```diff
@@ -33,6 +33,12 @@ var (
 		nil, nil,
 	)
 
+	pingdomMinRequestLimitDesc = prometheus.NewDesc(
+		"pingdom_rate_limit_min_remaining_requests",
+		"Tracks the minimum remaining requests allowed before hitting the short-term or long-term rate limit in the Pingdom API.",
+		nil, nil,
+	)
+
```

Comment:
I think the metric name ("minimum remaining requests") is confusing when we talk about limits. Maybe naming the gauge `pingdom_rate_limit_remaining_requests` (dropping the `min`) might be a better / less confusing name?

Reply:
Done